document_page_approval/models/document_page_history_workflow.py

```diff
@@ -78,6 +82,7 @@ def _can_user_approve_page(self):
             )
 
     def can_user_approve_this_page(self, page, user):
```

Comment:
@lmignon  I think @api.multi is not necessary here, because there is no iteration on self here. I